### PR TITLE
AGP-66: Panvala Launch Partnership

### PR DESCRIPTION
**Pull request by niran**
We're excited to have Aragon as part of Panvala. Aragon is a key part of the broad community of teams and projects in the Ethereum ecosystem that work on common infrastructure directly or fund that work through grants. Panvala is designed to acknowledge those teams and give them the tools to incentivize more teams to join them.

### CODE REVIEW

File: AGPs/AGP-F.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+the Aragon Association the authority to make transfers over time.
+
+## Amount of the transfer
+Up to $2,000,000.
+
+## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)
+Up to 24 monthly transfers of $83,333.33 each. Contributions can be made for up to three months
+at a time, but without allowing a month to pass after any of the individual
+scheduled contributions. These recurring transfers can be stopped at the
+discretion of either the Aragon Association or Aragon token holders.
```
Comment (john-light) [inline AGPs/AGP-F.md]:
```suggestion
discretion of either the Aragon Association or Aragon Network Token holders.
```

File: AGPs/AGP-F.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+In Panvala’s [first batch of grants](https://medium.com/@Panvala/twelve-grants-awarded-in-batch-one-of-panvala-token-grants-59b8df7422fe) released on February 1, Aragon One earned 300,000 PAN for their work on the [Voting Relay Protocol](https://forum.aragon.org/t/simple-voting-relay-protocol-optimistic-vote-tallying/473), a layer 2 protocol to scale token voting on Ethereum. Those tokens will be included in the initial token distribution when Panvala launches on the mainnet in August.
+
+10,000,000 PAN have been set aside to divide between Panvala Launch Partners, selected grant recipients who have committed to donate pan they earn or purchase from other grantees over the first two years of Panvala. Launch Partners receive voting power for fulfilling their pledge. This AGP proposes giving the Aragon Association permission to donate up to 2 million USD over two years to Panvala on behalf of Aragon One. The final amount will be chosen by the Aragon Association as they see fit.
```
Comment (john-light) [inline AGPs/AGP-F.md]: > This AGP proposes giving the Aragon Association permission to donate up to 2 million USD over two years to Panvala on behalf of Aragon One.

Is the AA actually donating actual USD or are they donating PAN that Aragon One and other grant recipients receive and then give back to the AA (presumably as some condition for receiving funding from the AA)?

Comment (niran) [reply AGPs/AGP-F.md]: Panvala's smart contracts can only handle PAN. The AA donates PAN earned by grant recipients to Panvala. During periods when the AA does not have enough PAN to fulfill the pledge, they use the treasury to acquire PAN to fulfill the pledge, or walk away from the pledge and the remaining tokens at the discretion of the AA or ANT holders.

**Comment by niran** [comment]
@john-light I'm unfamiliar with the process for moving this AGP to Stage IV, but I'm ready when you are. If you want the last comment to be addressed in the AGP itself, I can do that. I was trying to avoid going into more depth than necessary to explain the concept that will be unfamiliar to most readers.

**Pull request merged**